lib/stdlib/src/io_lib.erl

```diff
@@ -72,13 +72,17 @@ used for flattening deep lists.
 -compile(nowarn_deprecated_catch).
 
 -export([fwrite/2,fwrite/3,fread/2,fread/3,format/2,format/3]).
+-export([bfwrite/2, bfwrite/3, bformat/2, bformat/3]).
 -export([scan_format/2,unscan_format/1,build_text/1,build_text/2]).
 -export([print/1,print/4,indentation/2]).
 
--export([write/1,write/2,write/3,nl/0,format_prompt/1,format_prompt/2]).
--export([write_binary/3]).
+-export([write/1,write/2,write/3,write/5,bwrite/2,write_bin/5]).
```

Comment:
Should the undocumented ones such as `write_bin/5` have a separate `-export` statement?

Reply:
Yes